src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Nop, maybe I not explained myself... Current users(page developers) **don't** need to change anything. The new app developers that uses nw.js **and want to debug the node code** need to add the URL.

What I'm trying to say is that page developers **already** have the problem to put the "url" prop if they what to use chrome while developing pages otherwise, they risk the debugger to attach to the first tab. This behavior exist, and that filter will not avoid that.